guide/12-enrich-data-with-thematic-information/part7_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>Standard geographies are jurisdictional areas determined by government agencies. At the highest level, these are the countries of the world, and within these countries, the hierarchical levels have different names. If you already have a list of jurisdictional area identifiers, such as postal (ZIP) codes or U.S. Census Block Group Identifiers, they can be used directly as input to the <code>enrich</code> method to retrieve demographic information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>